### ISSUE TITLE
♿ 🐞 Bug Report: Full screen search widget is not keyboard accessible

### ISSUE THREAD

**Issue by aharonyltd** [elementor/elementor#19907]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The full screen search widget is not accessible
I start navigating the site with the keyboard with the TAB key

Once I get to the search icon:
1. He doesn't get focus
2. If you press ENTER on the keyboard, it moves to the search results page
3. The search form does not open in full screen as soon as you navigate to the search icon.

solution:
I think the user with the keyboard should be allowed to choose whether to open the search form or not
And it is important to have focus on the search icon

After the user has chosen with the ENTER key to open the search form, he will have the option to write the search query and press ENTER, and of course there will also be visual focus in the search form

Thank you

### Steps to reproduce

1. Build a simple website
2. Install the HELLO Elementor template

3. Build a citrus menu
4. Add a search widget
5. Set the search widget to full screen
6. Go incognito
7. Press TAB on the keyboard until you reach the search icon
8. You will see that it does not open and if you press ENTER it transfers to the search results page

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.7.38
	PHP Version: 7.4.30
	PHP Memory Limit: 128M
	PHP Max Input Vars: 4000
	PHP Max Post Size: 800M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.2
	Site URL: https://dev.arcb.co.il
	Home URL: https://dev.arcb.co.il
	WP Multisite: No
	Max Upload Size: 800 MB
	Memory limit: 40M
	Max Memory limit: 1508M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Asia/Jerusalem
	Admin Email: bellaharoni24@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: he_IL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36

== Active Plugins ==
	Asset CleanUp Pro: Page Speed Booster
		Version: 1.2.2.0
		Author: Gabe Livan

	Elementor
		Version: 3.7.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.7
		Author: Elementor.com

	Header Footer Code Manager Pro
		Version: 1.0.10
		Author: 99robots

	JetEngine
		Version: 3.0.4
		Author: Crocoblock

	ShortPixel Image Optimizer
		Version: 5.0.9
		Author: ShortPixel

	String Locator
		Version: 2.6.0
		Author: InstaWP

	Yoast SEO Premium
		Version: 13.5
		Author: Team Yoast

== Must-Use Plugins ==
	Asset CleanUp Pro: Plugin Filtering
		Version: 1.0
		Author: Gabriel Livan

== Elements Usage ==
	
	archive : 1
		breadcrumbs : 1
		container : 2
		divider : 1
		heading : 1
		nav-menu : 1
		posts : 1
	footer : 1
		container : 12
		heading : 9
		html : 1
		icon : 3
		icon-list : 5
		search-form : 1
		text-editor : 1
		theme-site-logo : 1
	header : 1
		container : 3
		icon-list : 1
		nav-menu : 1
		search-form : 1
		theme-site-logo : 1
	wp-page : 7
		breadcrumbs : 4
		button : 10
		container : 83
		divider : 12
		form : 2
		heading : 38
		html : 2
		icon-list : 2
		image : 3
		image-box : 15
		jet-listing-grid : 1
		nav-menu : 1
		posts : 1
		sitemap : 1
		slides : 1
		social-icons : 1
		testimonial-carousel : 1
		text-editor : 15
	single-post : 2
		breadcrumbs : 2
		container : 13
		gallery : 1
		heading : 6
		icon-list : 2
		nav-menu : 1
		post-comments : 1
		post-navigation : 1
		posts : 2
		share-buttons : 2
		table-of-contents : 1
		theme-post-content : 2
		theme-post-featured-image : 1
		theme-post-title : 1
	jet-listing-items : 0
		container : 4
		heading : 2
		image : 1

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Inactive
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	Import Export Template Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==
	
PHP: showing 3 of 3PHP: 2022-09-18 16:41:09 [notice X 42][/home/arcb/public_html/dev/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-09-18 16:44:38 [notice X 1][/home/arcb/public_html/dev/wp-content/plugins/elementor/core/common/modules/connect/module.php::230] Trying to get property 'email' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-09-21 21:40:22 [notice X 12][/home/arcb/public_html/dev/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::264] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 10 of 102022-09-21 15:43:31 [info] elementor::elementor_updater Started 
2022-09-21 15:43:31 [info] אלמנטור/Upgrades - _on_each_version Start  
2022-09-21 15:43:31 [info] אלמנטור/Upgrades - _on_each_version Finished 
2022-09-21 15:43:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]
2022-09-21 15:43:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]
2022-09-21 15:43:35 [info] elementor-pro::elementor_pro_updater Started 
2022-09-21 15:43:35 [info] אלמנטור פרו/Upgrades - _on_each_version Start  
2022-09-21 15:43:35 [info] אלמנטור פרו/Upgrades - _on_each_version Finished 
2022-09-21 15:43:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]
2022-09-21 15:43:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'אלמנטור פרו',
  'from' => '3.7.6',
  'to' => '3.7.7',
)]

JS: showing 2 of 2JS: 2022-09-21 16:01:18 [error X 2][https://dev.arcb.co.il/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.7:3:692080] T.getContainer is not a function 
JS: 2022-09-22 14:31:53 [error X 1][https://dev.arcb.co.il/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.7:3:225678] Cannot assign to read only property \'value\' of object \'#\' 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hello @aharonyltd ! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by libbeydesign** [comment]
Following

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
